config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/studios", to: "studios#index"
+
+  get "/movies/:id", to: "movies#show"
+
+  post "/movies/:id", to: "movies#create"
```

Comment:
Mostly good restful routes here, other than line 10 which I've already commented on above.

Reply:
Remember that REST is the combination between verb, path, and controller action.